ci/jenkins/test.sh

```diff
@@ -674,7 +673,7 @@ function run_e2e {
     if [[ $TESTBED_TYPE == "flexible-ipam" ]]; then
         go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus --antrea-ipam
     elif [[ $TESTBED_TYPE == "kind" ]]; then
-        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider kind -timeout=100m --prometheus
+        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider kind --kind.kubeconfig ${KUBECONFIG_PATH} -timeout=100m --prometheus
```

Comment:
Can you explain more about what's the failure of multiple kind job runs on same VM? I didn't see any details in the issue #6446
Another question is for `--kind.kubeconfig ${KUBECONFIG_PATH}`, not all test.sh in our CI jobs use the `--kubeconfig`, May I know what if a default `.kube/config` contains multiple contexts? How to guarantee that the default active one is the right one?

Reply:
The reason for using this is that the kind provider fetches the kubeconfig from the default kubeconfig location and sets the context to that, but if we have multiple clusters on the same vm then this will not work because for each run we would require to set the context to a different one.

Reply:
I doubt this fix will really work if you remove the line `kubectl config use-context kind-${KIND_CLUSTER}`.
I saw one given path is `${{PWD}}/.kube/config` in the the file `macros.yaml`, e.g: ` ./ci/jenkins/test.sh --testcase '{conformance_type}' --registry ${{DOCKER_REGISTRY}} --kubeconfig ${{PWD}}/.kube/config --testbed-type "kind" --kind-cluster-name "${{FULL_CLUSTER_NAME}}" --build-tag "${conformance_type}-${{BUILD_NUMBER}}"`

A kubeconfig file `${{PWD}}/.kube/config` may contain multiple contexts in one Jenkins Node. If you didn't set the context, how would you guarantee the right context is used?